jupyter_sphinx/execute.py

```diff
@@ -124,6 +131,8 @@ class JupyterCell(Directive):
         If provided, the cell output will not be displayed in the output.
     code-below : bool
         If provided, the code will be shown below the cell output.
+    no-thebelab : bool
```

Comment:
This seems an unnecessary complication that will see little use, let's remove it and reinstate if there is a request.

Reply:
While this specific implementation is not optimal and would not get used a lot, there should at least be _some_ way to not enable thebelab everywhere if you want to use it only at specific places/pages.

Reply:
Why should there be some way? Thebelab isn't visible before the user clicks the button, and so this doesn't seem to be a problem.

I suggest to wait for a use case because it's easy to implement this back, but having a specific application would allow us to make a more informed choice (maybe it makes more sense on the level of kernel) without running into backwards compatibility issues.

Reply:
I agree that it is a good idea to wait until we see a real life use case. Removed in 4cf2aec.